packages/framework/src/schemas/steps/channels/in-app.schema.ts

```diff
@@ -21,6 +21,7 @@ const redirectSchema = {
   if: {
     properties: {
       url: {
+        type: 'string',
```

Comment:
Needed to supress this strict validation warning:
<img width="491" alt="image" src="https://github.com/user-attachments/assets/f4cf3bd3-5418-4b93-8728-d706d50b8460">